sdk/internal/test/tracing/validator.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package tracing
```

Comment:
Since this works with `azcore/tracing`, maybe we should rename this package to something like `tracevalidator` or something like that so it doesn't collide with `azcore/tracing`. What do you think?

Reply:
I think that makes sense, I also had the same doubt but not sure if the best practice for the SDK is to keep the package naming short. I will update it to `tracingvalidator`

Reply:
I personally like the word `validator` a bit more haha, I think it fits better with what it's doing. But I am open to your preferences :)